source/extensions/omni.isaac.lab/omni/isaac/lab/assets/articulation/articulation_data.py

```diff
@@ -70,6 +70,9 @@ def __init__(self, root_physx_view: physx.ArticulationView, device: str):
         self._joint_acc = TimestampedBuffer()
         self._joint_vel = TimestampedBuffer()
 
+        # Link com
```

Comment:
NIT: Might be better to spell this one out or `CoM`
```suggestion
        # Link center of mass
```